webapp/crashstats/settings/base.py

```diff
@@ -528,58 +503,10 @@ def filter(self, record):
 # Set to True enable analysis of all model fetches
 ANALYZE_MODEL_FETCHES = config("ANALYZE_MODEL_FETCHES", True, cast=bool)
 
-# This `IMPLEMENTATIONS_DATABASE_URL` is optional. By default, the
-# implementation classes will use the config coming from `DATABASE_URL`.
-# For local development you might want to connect to different databases
-# for the Django ORM and for the socorro implementation classes.
-implementations_database_url = config("IMPLEMENTATIONS_DATABASE_URL", "")
-if not implementations_database_url:
-    implementations_database_url = database_url
-implementations_config = dj_database_url.parse(implementations_database_url)
-
 # The list of valid rulesets for the Reprocessing API
+# FIXME(willkg): we can pluck this from settings or structure
 VALID_RULESETS = ["default", "regenerate_signature"]
 
-# The CrashQueueBase class to use for submitting priority and reprocessing
```

Comment:
All the Socorro configuration bits are now in `socorro.mozilla_settings` which is accessed via `from socorro import settings as socorro_settings`. It's nice to have the settings in one place now.